src/main/java/ca/wolfram/beta/symath/operations/MultiplyNode.java

```diff
@@ -45,7 +91,15 @@ public double operationEval(VMap map) {
     public String toString() {
         StringBuilder s = new StringBuilder();
         s.append("(");
-        getChildren().forEach((c) -> s.append(c).append(" * "));
-        return s.substring(0, s.length() - 3) + ")";
+        boolean isFirst = true;
+        for (MathNode c : getChildren()) {
+            if (isFirst) {
+                s.append(c.toString());
+                isFirst = false;
+                continue;
+            }
+            s.append(" * ").append(c.toString());
+        }
```

Comment:
I think it is better to do 

```

StringBuilder s = new StringBuilder();
ListIterator<MathNode> iter = getChildren().iterator();
if (iter.hasNext) s.append("( ").append(iter.next().toString());
while( iter.hasNext) s.append("* ").append(iter.next().toString());
return s.append(")").toString();

```

Also i believe it is possible to have a MultiplyNode of 0 children. If that happens, then currently this function will return "()" and my style will return "(" we should make sure that multiplyNode can not have 0 children. If that is the case, we can also remove the `if` check in my code